src/main/java/dataObjects/Data.java

```diff
@@ -3,7 +3,11 @@
 import java.util.List;
 
 public class Data {
-    String labelType; // ["text", "line", "dot"...]
+    enum labelType {
```

Comment:
Ideally, the enum would be declared in a separate class. Additionally, after describing an enum, you'd still need to use it with a variable, so if you have an "enum Example {A,B,C}", to use it you would declare a variable "Example variableName;"